release_notes/Service_Fabric_ReleaseNotes_81.md

```diff
@@ -0,0 +1,111 @@
+    * The simplest mitigation, when possible, is to delete and recreate the application in SF 7.0.<br/>
+    * The other option is to upgrade the application in SF 7.0 (for example, with a version only change).<br/>
+
+    If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+
+
+## Upcoming Breaking Changes
+* .NET Core runtime LTS 2.1 runtime will go out of support from Aug 21, 2021. Service Fabric releases after that date will drop support for Service Fabric apps running with .NET Core 2.1 runtime. Service Fabric .NET SDK will take a dependency on .NET runtime 3.* features to support Service Fabric .NET Core apps.  This has no impact on Service Fabric .NET Framework SDK.
+
+
+## Service Fabric Runtime
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | [SF] Add support for Auxiliary Replica | **Brief desc**: In this release we are introducing support for a new kind of replica. These replicas are called 'Auxiliary Replicas', and their purpose is to enable advanced scenarios where an additional replica is necessary for specialized work required at replicator level that is not currently captured by the existing Roles of Primary, IdleSecondary, ActiveSecondary, None, and Unknown.<br>Auxiliary replicas will not be granted read or write access and Service Fabric will not consider* them as potential targets during Primary elections due to balancing or failover. This opens new opportunities like having a replica that witnesses the replication stream to perform monitoring or auditing, or to act as a temporary cache or transaction log that can be reused by other replicas.<br>One of the applications for auxiliary replicas is assigning them fewer cluster resources since they have fewer duties. This means that in some cases Services can consume fewer resources while maintaining roughly the same levels of reliability, reducing COGS.<br>To use this new feature, the cluster level configuration 'EnableAuxiliaryReplicas' must be set to true and a stateful persisted service would need to be created or updated with the 'AuxiliaryReplicaCount' value, which indicates the number of auxiliary replicas per partition.<br>This is feature is targeted for advanced users/scenarios and require corresponding replicator changes. The default Service Fabric Replicator and Reliable Collections stack does not rely on this, though it may be augmented in future to take advantage of it. Please reach out to Service Fabric team for more details.<br>* Auxiliary replicas can be promoted to primary role under certain scenarios, but this transition is transient, and the replica will not be granted read or write status.<br>**Usage**: New capability in Service Fabric<br>**Impact**: Impacts stateful persisted service using auxiliary replicas.<br> |
```

Comment:
not consider* - Bold? 

Any documentation for how to configure auxiliary replicas?

Reply:
The * here s a footnote marker with a footnote below.
The Aux replica is an advanced feature that requires a custom replicator and hence may not have detailed documentation, but Manoj is working on some documentation for it.